src/modules/Dropdown/Dropdown.js

```diff
@@ -385,10 +385,10 @@ export default class Dropdown extends Component {
 
   selectHighlightedItem = (e) => {
     const { multiple, onAddItem, options } = this.props
-    const value = _.get(this.getSelectedItem(), 'value')
+    const { value, disabled } = this.getSelectedItem() || {}
```

Comment:
Am I correct in thinking you cannot actually highlight (select) a disabled item due to the [logic added](https://github.com/TechnologyAdvice/stardust/pull/482/files#diff-609214949026e594dbfd98481664c928R648) to the `moveSelectionBy`?

***

It's beyond the scope of this PR, but It could be this method name needs updated.  `selected` means highlighted, `active` means it is part of the value.  `selectHighlightedItem` is a little redundant as the highlighted item _is_ already selected.  It might be more accurately named `makeSelectedItemActive` or similar.